pipeline/filter/check_alerts_areas.py

```diff
@@ -52,7 +52,12 @@ def check_alerts_against_area(alertlist, area):
     alertdelist  = alertlist['de']
 
     # here is the crossmatch
-    result = area['moc'].contains(alertralist*u.deg, alertdelist*u.deg)
+    try:
+        result = area['moc'].contains(alertralist*u.deg, alertdelist*u.deg)
+    except Exception as e:
+        log = lasairLogging.getLogger("filter")
```

Comment:
Side note: We should think about a way to easily expose these logs to the dev team. @gpfrancis do you know of a widget/plugin for grafana that could present error and warning logs?